cyipopt/scipy_interface.py

```diff
@@ -59,18 +58,20 @@ def __init__(self, fun, args=(), kwargs=None, jac=None, hess=None, hessp=None,
         self._constraint_funs = []
         self._constraint_jacs = []
         self._constraint_args = []
+        self._constraint_kwargs = []
```

Comment:
What's the purpose of adding this and the other lines with `con_kwargs`? It seems unrelated to the fix.